# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glib/no-xlocale.patch.cross0
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glib-2.64.1/meson.build.vanilla	2020-03-15 12:15:55.884281368 +0100
+++ glib-2.64.1/meson.build	2020-03-15 12:16:03.276281371 +0100
@@ -274,7 +274,7 @@
   'unistd.h',
   'values.h',
   'wchar.h',
-  'xlocale.h',
+  '_noxlocale.h',
 ]
 
 foreach h : headers
